### ISSUE TITLE
Improve drawing order (Z-axis) of recordings

### ISSUE THREAD

**Issue by SokoFromNZ** [andreynovikov/trekarta#276]
I run Trekarta on a Nokia 2.3 (very cheap phone) with almost 300 recordings on it. On start it takes 20 seconds for all recordings to load and draw.
When starting the recording the red line for the current/recording track is drawn above all the existing recordings (I've changed all their color to blue).

On a long bike ride (>3 hours) Trekarta crashes regularly (fyi only since the last couple of versions where you improved the UI). When starting it again the recoring continues... so far so good!
But the line for the current/recording track is then drown under the existing recordings and so I cannot see it anymore as I often ride the same paths.

As I cannot reproduce the crashes - I reckon its due to the crappy phone with low ram - it would help a lot if you can put the current track on top of the existing one once they are loaded after a crash.

thx
Soko

**Comment by andreynovikov** [comment]
Does it ask you to send crash report when you restart Trekarta after crash? (Do you know you can switch off unnecessary recordings by pressing eye icon?)

**Comment by SokoFromNZ** [comment]
No, it does not ask to send a crash report. As far as I remember... the last time it happened... the foreground service/notification was still running. But the main window closed. After reopening Trekarta it loaded all existing recordings again.

I know about the eye icon. Bdw: is it possible to use an inversion of it? I dont' want to click on 300 eye icons to hide/show them.

Anyhow. I do wanna see all 300 recordings anyhow so I know which roads I haven't biked on yet.

**Comment by andreynovikov** [comment]
Then it looks like Android closes (kills) the app to free memory.

**Comment by SokoFromNZ** [comment]
Yeah, thats my guess too. I have the same issue with an app of mine which is always on the edge of RAM on 4GB devices. There is nothing you can do about it... besides finding memory leaks of course ;)

Anyhow... I'm not sure how you determine on app start if a recording is still running... but in that case it would help if you can put the current recording-line on top of the loaded/existing lines.

**Comment by andreynovikov** [comment]
Done it already: 73f9812076c31ab4b82231d86f65f6c8864896c2
Will be published soon
